.github/workflows/ci.yml

```diff
@@ -107,6 +107,11 @@ jobs:
             run-tests: 'yes'
             collect-code-coverage: 'yes'
 
+          - operating-system: 'ubuntu-20.04'
+            php-version: '8.3'
```

Comment:
```suggestion
            php-version: ${{ env.PHP_MAX }}
```

Reply:
I had to revert this change as it was causing problem - none of CI jobs were added to the workflow. Maybe it should be in single quotes or without `${{ }}`, I don' know, but coverage job also is run only on 8.3 and has this version explicitly defined in the matrix. If we want to clean it up, we can do it later.